apps/docs/pages/guides/auth/auth-helpers/nextjs-server-components.mdx

```diff
@@ -95,7 +95,7 @@ Create a new file at `/utils/supabase-browser.ts` and populate with the followin
 
 ```ts
 import { createBrowserSupabaseClient } from '@supabase/auth-helpers-nextjs'
-import { Database } from '../db_types'
+import { Database } from '../lib/database.types'
```

Comment:
The [CLI doc](https://supabase.com/docs/reference/javascript/typescript-support) recommends generating types into `lib/database.types.ts`, which are referenced to in [other docs](https://supabase.com/docs/guides/auth/auth-helpers/nextjs) that link to this doc, resulting in a slight discrepancy.